File: Tests/FuzzyEqualsTests.fs
namespace Extensions

open System
open Stringier
open Microsoft.VisualStudio.TestTools.UnitTesting

[<TestClass>]
type FuzzyEqualsTests() =
    [<DataTestMethod>]
    [<DataRow(true, "bob", "bob")>]
    [<DataRow(true, "bob", "mob")>]
    [<DataRow(false, "bob", "mom")>]
    member _.``fuzzyEquals string string`` (exp:bool, src:string, oth:string) =
        Assert.AreEqual(exp, src |> fuzzyEqual oth)

File: Tests/FuzzyEqualsTests.cs
using System;
using Xunit;

namespace Langly {
	public class FuzzyEqualsTests {
		[Theory]
		[InlineData("bob", "bob", true)]
		[InlineData("bob", "mob", true)]
		[InlineData("bob", "mom", false)]
		public void FuzzyEquals(String source, String other, Boolean expected) => Assert.Equal(expected, Text.FuzzyEquals(source, other));
	}
}
